react-components/src/components/RevealToolbar/SlicerButton.tsx

```diff
@@ -0,0 +1,115 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type ReactElement, useState, useEffect } from 'react';
+
+import { Box3, Plane, Vector3 } from 'three';
+
+import { useReveal } from '../RevealContainer/RevealContext';
+import { Button, Dropdown, Menu, RangeSlider } from '@cognite/cogs.js';
+
+import styled from 'styled-components';
+
+export const SlicerButton = (): ReactElement => {
+  const viewer = useReveal();
+
+  const [{ minHeight, maxHeight, topRatio, bottomRatio }, setSliceState] = useState<{
+    minHeight: number;
+    maxHeight: number;
+    topRatio: number;
+    bottomRatio: number;
```

Comment:
These can be `type`
`type SliceState = {`
  `minHeight: number;`
  `maxHeight: number;`
  `topRatio: number;`
  `bottomRatio: number;`
`}`